libs/authorizenet/driver/in-memory/src/drivers/authorize-net-driver.module.ts

```diff
@@ -23,6 +27,10 @@ export class DaffAuthorizeNetInMemoryDriverModule {
           provide: DaffAuthorizeNetDriver,
           useExisting: DaffInMemoryAuthorizeNetService,
         },
+        {
+          provide: DaffAuthorizeNetPaymentId,
+          useValue: MAGENTO_AUTHORIZE_NET_PAYMENT_ID,
```

Comment:
I think you're using the wrong string CONST here

Reply:
fixed